client/flutter/lib/components/page_scaffold/page_header.dart

```diff
@@ -23,33 +23,40 @@ class PageHeader extends StatelessWidget {
       leading: Container(),
       expandedHeight: 120,
       backgroundColor: Colors.white,
-      flexibleSpace: FlexibleSpaceBar(background:_buildHeader()),
+      flexibleSpace: FlexibleSpaceBar(background: _buildHeader()),
     );
   }
 
   Widget _buildHeader() {
     List<Widget> headerItems = [
-      if (this.showBackButton) BackArrow(),
-      
-      Column(
+      if (this.showBackButton)
+        Transform.translate(offset: Offset(-12, 0), child: BackArrow()),
```

Comment:
Why this offsetting to the left? 🤔

Reply:
The back button svg icon has extremely too much padding. I moved it to align it where the back button is expected to be. See the photos. 

It should be a better fix for this. But didn’t check the svg viewport yet

Reply:
I didn't check it into much details, but this change impacts all headers. So I'll let somebody else decide 👍 
FYI: @patniemeyer @advayDev1 @theswerd